lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -476,7 +476,7 @@ def _kill_dhclient(self, if_name: str, af_inet: int):
 
         # Kill dhclient if running.
         pgrep_out = subprocess.Popen(
-            ["pgrep", "-f", "dhclient.*" + if_name],
+            ["pgrep", "-f", "^/sbin/dhclient.*" + if_name],
```

Comment:
Are we sure that `/sbin/dhclient` is used for starting the processes? On my host:
```
sudo dhclient docker0
---
$ pgrep -fa dhclient.*docker0
1487058 sudo dhclient docker0
1487059 dhclient docker0
1487060 dhclient docker0
```
This is, `$ pgrep -f ^/sbin/dhclient.*docker0` is empty for me.

Reply:
On AGW dhclient is started by sys init which uses above signature.